src/components/customCharts/SimpleBarChart.tsx

```diff
@@ -26,21 +30,26 @@ const useStyles = makeStyles(theme => ({
 }));
 
 const CHART_PADDING = 10;
+const CHART_DETAILS_COLOR = 'rgb(72,72,72)';
 
-const handleElementClick = (setClickedCallback: any, objects: any, clickPositions: any) => {
+const handleElementClick = (
+  setClickedCallback: any,
```

Comment:
Tu już chyba może być typ Function? (Ale to uwaga na przyszłość ;) )

Reply:
a no powinno, jakoś przeoczyłem bo przynajmniej takie oczywiste staram się odrazu dobrze oznaczać 😉 zaraz przejrzę typy jeszcze raz i poprawię